com.adobe.Flash-Player-Projector.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "com.adobe.Flash-Player-Projector",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "flashplayer",
+    "separate-locales": false,
```

Comment:
Why? i don't think it has any effect when using `extra-data`.

Reply:
The spotify JSON file was using that as well. I've removed it.